modules/packages/LinearAlgebra.chpl

```diff
@@ -444,16 +444,19 @@ proc eye(Dom: domain(2), type eltType=real) {
 proc setDiag (ref X: [?D] ?eltType, in k: int = 0, val: eltType = 0)
               where isDenseMatrix(X) {
   var start, end = 0;
+  var (low_i, low_j) = D.low;
+  var (high_i, high_j) = D.high;
   if (k >= 0) { // upper or main diagonal
-    start = 1;
-    end = D.shape(1) - k;
+    start = low_i;
+    end = high_i - k;
```

Comment:
Let's change all `low_i`-like variable names to camelCase. e.g. `lowI`, `highI`... Personally, that looks ugly to me and I'd prefer `iLow`, `iHigh` instead.